test/ClientHttpBinIntegrationTest.php

```diff
@@ -21,17 +21,17 @@
 use Amp\Http\Rfc7230;
 use Amp\Http\Server\Options;
 use Amp\Http\Server\Request as ServerRequest;
-use Amp\Http\Server\RequestHandler\CallableRequestHandler;
+use Amp\Http\Server\RequestHandler\ClosureRequestHandler;
 use Amp\Http\Server\Response as ServerResponse;
 use Amp\Http\Server\Server;
 use Amp\PHPUnit\AsyncTestCase;
+use Amp\Pipeline\Pipeline;
 use Amp\Socket;
+use Closure;
```

Comment:
We usually don't import global symbols but directly reference them using `\Closure` instead.